includes/apple-exporter/components/class-table.php

```diff
@@ -358,6 +304,14 @@ protected function build( $html ) {
 			'default-table',
 			'default-table'
 		);
+
+		// Register dark mode styles, if applicable.
+		if ( $this->dark_table_colors_exist ) {
+			$this->register_component_style(
+				'dark-table',
+				'dark-table'
+			);
+		}
```

Comment:
Created `$this->dark_table_colors_exist` so that the bool determination only has to be made once in the `register_specs` method but is still available here in the `build` method where the component_styles are registered.